README.md

```diff
@@ -185,6 +202,17 @@ The official way to find the ECR image URIs for your region is to use the SSM Pa
 aws ssm get-parameters-by-path --path /aws/service/aws-for-fluent-bit/
 ```
 
+#### Networking issue with Windows containers when using `async` DNS resolution by plugins
```

Comment:
I think we need this but it does not fit here under the image distribution section. 

Instead, I would prefer this be added as a section in the debugging.md and then link it under the main tag in the readme for windows support.

Reply:
Let's create a windows specific section of the debugging guide and then link to it and specifically call out that the guidance must be followed if you run windows containers in default namespace

Reply:
I mean in the readme say "Check out the Windows debugging guide. There is a known issue with..." something like that

Reply:
windows section of the debugging guide